docs/development/deployment/chain-spec.md

```diff
@@ -2,24 +2,151 @@
+this information must be established at the outset of the chain and cannot be changed thereafter
+without starting an entirely new blockchain. There are no command-line flags to override values in
+the genesis portion of a chain spec.
 
-* What is a Raw Spec?
+Examples of what information might be included in the genesis portion of a chain spec include
+initial token balances, the accounts that are initially part of a governance council, or the holder
+of the sudo key. Substrate nodes also place the compiled Wasm runtime logic on chain, so the initial
+runtime must also be supplied in the chain spec.
+
+It is this second part of the chain spec that is used when creating a genesis block.
+
+## Storing Chain Spec Information
+
+The information that comprises a chain spec can be stored in either of two ways. Being a Rust
+struct, the first way to store this information is as Rust code. Indeed, Substrate nodes typically
+include at least one, and often many, chain specs hard-coded into the client. Including this
+information directly in the client ensures that the node will know how to connect to at least one
+chain without any additional information supplied by the node operator. In protocols that have a
+notion of "main net" this spec is usually hard-coded in the client.
+
+Another common way to store chain spec information is in json format. The chain spec struct has a
+method for serializing its data into JSON as well as a function for de-serializing JSON data into an
+instance of a chain spec. When launching testnets or private chains, it is common to distribute a
+JSON-encoded chain spec along with the node binary.
+
+## Using Chain Specs
+
+Node operators and runtime developers will encounter chain specifications when performing common
+tasks.
+
+### Launching a Chain
+
+Each time a node operator starts a node, they provide a chain specification that the node should
+use. In the simplest case, the chain spec is provided implicitly and the node uses a default chain
+spec that is hard-coded into the node binary.
+
+A common task is to start a testnet or private network that behaves similarly to an existing
+protocol, but is not connected to the mainnet. To achieve this, the operator may choose an
+alternative hard-coded chain spec by using a command-line flag such as `--chain local` that
+instructs the node to use the spec associated with the string "local". A third option available to
+node operators is to provide a chain spec as a JSON file with a command-line flag such as
+`--chain=someCustomSpec.json`, in which case the node will attempt to de-serialize the provided JSON
+chain spec, and then use it.
+
+### Developing a Runtime
+
+Nearly every Substrate runtime will have storage items that need to be configured at genesis. When
+developing with [FRAME](../../conceptual/runtime/frame.md), any storage item that is declared with
+the `config()` option requires configuration at genesis. It is the job of the chain spec,
```

Comment:
We could add a comprehensible example on how to use the `config()`. It may not be required for the current scope; however, it will be a common question.

Reply:
Log an issue for it, maybe that belongs in recipes?